Dockerfile

```diff
@@ -44,8 +44,6 @@ COPY --from=build --chown=1000:1000 \
     /app/support/openapi.yaml \
     /app/support/schema.json \
     /app/support/live-templates.txt \
-    /app/support/render-data-asset-diagram.sh \
```

Comment:
I'd say this is the most controversial of my changes. Idea here is to avoid having this in a script and inline into the code (see changes for main.go)